app/controllers/application_controller.rb

```diff
@@ -5,6 +5,9 @@ class ApplicationController < ActionController::Base
 
   before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :previous_url_for_cookies_page
+  before_action :check_privacy_policy_accepted
+
+  private
```

Comment:
This cannot go here. `check` needs to be public

Reply:
This is why your deploy is failing

Reply:
Doh! Fixed!